base/batch/cronjobs/okd-coreos-4.16-prod-create-daily.yaml

```diff
@@ -39,7 +39,7 @@ spec:
               - name: SINK_URL
                 value: "http://el-okd-coreos-prod.okd-coreos.svc.cluster.local:8080"
               - name: BRANCH
-                value: "release-4.16"
+                value: "release-4.17"
```

Comment:
should we set master?

Reply:
i thought about it..but let's say 4.17 diverges from master and 4.18 becomes master (hopefully by then we won't need this pipeline), we would need this to be in sync with 4.17?

Reply:
@travier could you recommend what we want to set here? we can set it to master, but I am worried about the divergence, but hopefully we don't need to worry that far because scos images are around the corner?

Reply:
updated based on discussion here: https://redhat-internal.slack.com/archives/C06QQ74Q2DB/p1720803008774029